ush/wave_prnc_cur.sh

```diff
@@ -75,22 +74,15 @@ ${NLN} "${DATA}/mod_def.${WAVECUR_FID}" ./mod_def.ww3
 export pgm="${NET,,}_ww3_prnc.x"
 source prep_step
 
-"${EXECgfs}/${pgm}" 1> prnc_${WAVECUR_FID}_${ymdh_rtofs}.out 2>&1
-export err=$?; err_chk
-if [ "$err" != '0' ]
-then
-  cat prnc_${WAVECUR_FID}_${ymdh_rtofs}.out
-  set $setoff
-  echo ' '
-  echo '******************************************** '
-  echo "*** WARNING: NON-FATAL ERROR IN ${pgm} *** "
-  echo '******************************************** '
-  echo ' '
-  set $seton
-  echo "WARNING: NON-FATAL ERROR IN ${pgm}."
-  exit 4
+"${EXECgfs}/${pgm}" 1> "prnc_${WAVECUR_FID}_${ymdh_rtofs}.out" 2>&1
+export err=$?
+err_chk
+if [[ "${err}" -ne 0 ]]; then
+    cat "prnc_${WAVECUR_FID}_${ymdh_rtofs}.out"
+    echo "WARNING: NON-FATAL ERROR IN ${pgm}."
+    exit 4
 fi
```

Comment:
@JessicaMeixner-NOAA Should this be a fatal error or not?  `err_chk` is going to exit completely if `err` is not 0, so the `WARNING` will never be printed.